katdal/sensordata.py

```diff
@@ -874,7 +874,9 @@ def get(self, name, select=False, extract=True, **kwargs):
                 # Otherwise, iterate through virtual sensor templates and look for a match
                 for pattern, create_sensor in self.virtual.items():
                     # Expand variable names enclosed in braces to the relevant regular expression
-                    pattern = re.sub(r'({[a-zA-Z_]\w*})', lambda m: '(?P<' + m.group(0)[1:-1] + '>[^//]+)', pattern)
+                    # (match anything but slashes, which are preferred delimiters in virtual sensor names)
+                    pattern = re.sub(r'({[a-zA-Z_]\w*})',
```

Comment:
Maybe escape the curly braces (from regex). Python tends to be getting stricter about escaping over time, so at some stage it may start complaining that the value inside the braces is not a valid repetition count.

Reply:
Good point, will do.

Reply:
Done.